msg/tools/generate_microRTPS_bridge.py

```diff
@@ -230,8 +230,15 @@ def check_rtps_id_uniqueness(classifier):
 # get FastRTPS version (major.minor, since patch is not relevant at this stage)
 fastrtps_version = ""
 try:
-    fastrtps_version = float(subprocess.check_output(
-        "ldconfig -v | grep libfastrtps | tail -c 6", shell=True).decode("utf-8").strip()[-5:-2])
+    version = subprocess.check_output(
+        "ldconfig -v | grep libfastrtps", shell=True).decode("utf-8").strip().split('so.')[-1]
+    version = [int(v) for v in version.split('.')]
+
+    # convert version string to float (in case of 1.10, 1.91 temporarily)
+    fastrtps_version  = 0.0
```

Comment:
```suggestion
    fastrtps_version  = float("{0:.2f}".format(version))
```
No need for the lines bellow.